src/index.js

```diff
@@ -16,6 +18,7 @@ const install = /*#__PURE__*/ installFactory({ plugins: { componentsPlugin, dire
 // BootstrapVue plugin
 const BootstrapVue = /*#__PURE__*/ {
   install: install,
+  NAME: NAME,
```

Comment:
Why not just `NAME` as below?

Reply:
Below is actually an export of variable names (named exports), not an export of an object with vars inside it